test/fixtures/mson-inheritance.json

```diff
@@ -12,11 +12,15 @@
         {
           "element": "object",
           "meta": {
-            "id": "User"
+            "id": "User",
+            "description": "The User description\n\n"
```

Comment:
Is the plan for serializers to account for these extra line breaks?  Seems like if you go from API Blueprint to Refract then back to API Blueprint, you may get extra spaces.